.eslintrc.js

```diff
@@ -1,12 +1,17 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
-
 const LICENSE_HEADER = `
 /*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
+ *   Copyright OpenSearch Contributors
```

Comment:
If both versions of license header are OK then I'd recommend keeping the license header as it was so that less change will be introduced in this PR.

Reply:
+1